authorization/invitation/service/invitation_service.go

```diff
@@ -391,25 +396,39 @@ func (s *invitationServiceImpl) Accept(ctx context.Context, token uuid.UUID) (st
 
 			err = s.Repositories().IdentityRoleRepository().Create(ctx, ir)
 			if err != nil {
-				return resourceID, err
+				return resourceID, redirectPath, err
+			}
+		}
+
+		resource, e := s.Repositories().ResourceRepository().Load(ctx, inviteToIdentity.IdentityResourceID.String)
+		if e != nil {
+			return resourceID, redirectPath, err
+		}
+
+		if resource.ResourceType.Name == authorization.IdentityResourceTypeTeam {
+			if resource.ParentResourceID != nil {
+				redirectPath, e = s.getRelativeSpacePath(ctx, *resource.ParentResourceID)
+				if e != nil {
+					return resourceID, redirectPath, err
+				}
 			}
 		}
 
 		// Delete the invitation
 		s.Repositories().InvitationRepository().Delete(ctx, inv.InvitationID)
 
-		// Return the identity ID
-		return inviteToIdentity.IdentityResourceID.String, nil
+		// Return the identity ID and redirect Path
+		return inviteToIdentity.IdentityResourceID.String, redirectPath, nil
```

Comment:
should it be `inviteToIdentity.IdentityResourceID.String()` (ie, a call to a method?)

Reply:
no. we don't have any method by name `String()` for NullString.
```go
type NullString struct {
	String string
	Valid  bool // Valid is true if String is not NULL
} 
```